src/service/CinemaService.java

```diff
@@ -0,0 +1,178 @@
+package service;
+
+import exception.AlreadyTakenSeatException;
+import exception.NonExistingSeatException;
+
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.stream.IntStream;
+
+public class CinemaService {
+
+    private static final String availableSeat = "S";
+    private static final String soldSeat = "B";
+    private final Scanner scanner;
+    private final Cinema cinema;
+
+    public CinemaService() {
+        scanner = new Scanner(System.in);
+        cinema = initializeCinema();
+        createSeats();
+    }
+
+    private Cinema initializeCinema() {
+        System.out.println("Enter the number of rows:");
+        int rows = scanner.nextInt();
+
+        System.out.println("Enter the number of seats in each row:");
+        int cols = scanner.nextInt();
+
+        System.out.println();
+
+        return new Cinema(rows, cols);
+    }
+
+    private void createSeats() {
+        String[][] seats = new String[cinema.getRows()][];
+
+        for (int i = 0; i < cinema.getRows(); i++) {
+            String[] row = new String[cinema.getColumns()];
+            Arrays.fill(row, availableSeat);
+            seats[i] = row;
+        }
+
+        cinema.setSeats(seats);
+    }
+
+    public void showMenu() {
+        while (true) {
+            System.out.print("""
+                    1. Show the seats
+                    2. Buy a ticket
+                    3. Statistics
+                    0. Exit
+                    """);
+
+            String input = scanner.next();
+            System.out.println();
+
+            switch (input) {
+                case "0" -> {
+                    scanner.close();
+                    return;
+                }
+                case "1" -> showSeats();
+                case "2" -> buyTicket();
+                case "3" -> statistics();
+                default -> System.out.printf("Unknown menu item - %s\n", input);
+            }
+        }
+    }
+
+    private void showSeats() {
+        System.out.print("Cinema:\n ");
+        IntStream.rangeClosed(1, cinema.getSeats()[0].length)
+                .forEach(i -> System.out.printf(" \t%d", i));
```

Comment:
Can you remove spaces after `Cinema:\n` and before `\t%d`?

Reply:
Fixed